linode_api4/objects/object_storage.py

```diff
@@ -436,6 +452,13 @@ class ObjectStorageCluster(Base):
         "static_site_domain": Property(),
     }
 
+    @deprecated(
```

Comment:
We may need to include `deprecated` in the requirements list 👍

Reply:
I think it has already been added into `pyproject.toml` in a previous PR.
https://github.com/linode/linode_api4-python/blob/bcf1cc5f019e0043acc73268a43b6294135acfcc/pyproject.toml#L34

Reply:
Oh oops, I missed that somehow lol. LGTM